client/extensions/woocommerce/woocommerce-services/views/label-settings/label-settings.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { getPaperSizes } from '../../lib/pdf-label-utils';
+import Button from 'components/button';
+import FormFieldSet from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormSelect from 'components/forms/form-select';
+import PaymentMethod from './label-payment-method';
+
+const ShippingLabels = ( { isLoading, paymentMethods, setFormDataValue, selectedPaymentMethod, paperSize, storeOptions, translate } ) => {
+	const onPaymentMethodChange = ( value ) => setFormDataValue( 'selected_payment_method_id', value );
+	const onPaperSizeChange = ( event ) => setFormDataValue( 'paper_size', event.target.value );
+
+	const renderPaymentMethod = ( method, index ) => {
+		const onSelect = () => onPaymentMethodChange( method.payment_method_id );
+
+		return (
+			<PaymentMethod
+				key={ index }
+				selected={ selectedPaymentMethod === method.payment_method_id }
+				type={ method.card_type }
+				name={ method.name }
+				digits={ method.card_digits }
+				expiry={ method.expiry }
+				onSelect={ onSelect } />
+		);
+	};
+
+	const renderPlaceholder = () => (
+		<div className="label-settings__placeholder">
+			<FormFieldSet>
+				<FormLabel className="label-settings__cards-label">
+					<span />
+				</FormLabel>
+				<FormSelect />
+			</FormFieldSet>
+			<FormFieldSet>
+				<FormLabel
+					className="label-settings__cards-label">
+					<span />
+				</FormLabel>
+				<p className="label-settings__credit-card-description">
+				</p>
+				<PaymentMethod selected={ false } isLoading={ true } />
+				<PaymentMethod selected={ false } isLoading={ true } />
+				<Button href="https://wordpress.com/me/billing" target="_blank" compact />
+			</FormFieldSet>
+		</div>
+	);
+
+	const renderContent = () => {
+		if ( isLoading ) {
+			return renderPlaceholder();
+		}
+
+		const paperSizes = getPaperSizes( storeOptions.origin_country );
+
+		return (
+			<div>
+				<FormFieldSet>
+					<FormLabel
+						className="label-settings__cards-label">
+						{ translate( 'Paper size' ) }
+					</FormLabel>
+					<FormSelect onChange={ onPaperSizeChange } value={ paperSize }>
+						{ Object.keys( paperSizes ).map( ( size ) => (
+							<option value={ size } key={ size }>{ paperSizes[ size ] }</option>
+						) ) }
+					</FormSelect>
+				</FormFieldSet>
+				<FormFieldSet>
+					<FormLabel
+						className="label-settings__cards-label">
+						{ translate( 'Credit card' ) }
+					</FormLabel>
+					<p className="label-settings__credit-card-description">
+						{ translate( 'Use your credit card on file to pay for the labels you print or add a new one.' ) }
+					</p>
+					{ paymentMethods.map( renderPaymentMethod ) }
+					<Button href="https://wordpress.com/me/billing" target="_blank" compact>
+						{ translate( 'Add another credit card' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **39** times:
`translate( 'Add Credit Card' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).